web/profiles/joinup/config/install/views.view.collections.yml

```diff
@@ -486,8 +482,9 @@ display:
       relationships: {  }
       arguments: {  }
       display_extenders: {  }
+      show_admin_links: false
```

Comment:
I also turned off the contextual links while I was at it, this added an additional cache context to the render array.